database_admin/schema/create_schema.sql

```diff
@@ -223,11 +222,24 @@ BEGIN
     DELETE
     FROM advisory_account_data
     WHERE (advisory_id, rh_account_id) NOT IN (SELECT advisory_id, rh_account_id FROM current_counts)
-      AND (advisory_id = advisory_id_in OR advisory_id_in IS NULL)
+      AND (advisory_id = ANY (advisories_id_in) OR advisories_id_in IS NULL)
```

Comment:
And probably we can remove in go code advisory_account_data locking as well. https://github.com/RedHatInsights/patchman-engine/blob/f166a58df39bda44339d8e9ec4d13e0aaaf6a9b9/evaluator/evaluate.go#L346

Reply:
We still need locking. If we have 2 concurrent evaluations and one calculates the caches while other modifies source data in system_advisories, we'll have inconsistencies. We need to lock the rows in advisory_account_data to prevent this from hapenning.